src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -20,7 +17,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal sealed class TupleTypeSymbol : NamedTypeSymbol, ITupleTypeSymbol
     {
-        private readonly NamedTypeSymbol _underlyingType;
+        private readonly OneOrMany<NamedTypeSymbol> _underlyingTypeChain;
```

Comment:
Is ```_underlyingTypeChain[0]``` the constructed type for ```ValueTuple<T1, ..., T7, ValueTuple<T8, ..., T15, ValueTuple<...>>>```? If so, doesn't that single constructed type hold the chain of underlying types?